localization/english/interfaces_l_english.yml

```diff
@@ -1801,7 +1801,7 @@
  CULTURE_TYPE_DESCRIPTION:0 "Culture, Era [Era.GetNumber]"
  ALREADY_ACQUIRED_BUTTON:0 "See Known Technologies"
 
- TECHNOLOGY_ACQUIRED_POPUP_HEADER:1 "#header [PlayerMessageItem.GetScopeTechnology.GetName] Unlocked#!"
+ TECHNOLOGY_AQUIRED_POPUP_HEADER:1 "#header [PlayerMessageItem.GetScopeTechnology.GetName] Unlocked#!"
```

Comment:
The related section in `.gui` file should be updated to reflect the correct spelling: `Acquired`

Reply:
Its in vanilla - gui file isn't present in mod

Reply:
Added vanilla file as to fix typo anyway